lib/resque/failure/extra.rb

```diff
@@ -79,7 +79,7 @@ def _each_with_index(opts={})
         end # while
       end
 
-      def each(*args,&block)
+      def custom_each(*args,&block)
```

Comment:
Something like `iterate_failures` seems more appropriate to me!

Reply:
Fixed!